presentation/details/src/main/java/com/giraffe/presentation/details/screens/castCredit/CastCreditScreen.kt

```diff
@@ -48,22 +52,39 @@ fun CastCreditScreen(
 @Composable
 private fun CastCreditContent(
     state: CastCreditScreenState,
-    interaction: CastCreditInteractionListener,
-    modifier: Modifier = Modifier
+    interaction: CastCreditInteractionListener
 ) {
-    BaseScreen(
-        title = stringResource(R.string.best_of_, state.actorName),
+    ScreenStates(
         isLoading = state.isLoading,
         isNoInternet = state.isNoInternet,
-        onRetryClick = interaction::onRetryClick,
-        onBackClick = interaction::onBackClick
+        onRetryClick = interaction::onRetryClick
     ) {
-        Box(modifier = modifier) {
-            TransitionBetweenColumnAndVerticalGrid(
-                poster = state.posters,
-                isListSelected = !state.isGridSelected,
-                onPosterClicked = interaction::onPosterClick
-            )
+        Box {
+            Column(
+                modifier = Modifier
+                    .background(Theme.color.background.screen)
+                    .fillMaxSize()
+                    .statusBarsPadding()
+            ) {
+                AppBar(
+                    title = stringResource(R.string.best_of_, state.actorName),
```

Comment:
When the language is Arabic, the app crashes with IllegalFormatConversionException. Change the title value and remove `R.string.best_of_` from the string resources, since we already have `R.string.best_of`.
```suggestion
                    stringResource(R.string.best_of) + " " + state.actorName,
```